selftests/isort.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/sh -e
 
-isort --recursive --check-only \
-      --skip selftests/.data/loader_instrumented/dont_crash.py \
-      --skip selftests/.data/loader_instrumented/double_import.py \
-      --skip selftests/.data/loader_instrumented/dont_detect_non_avocado.py \
-      --skip avocado/__init__.py
+PARENT=$(cd ..; pwd)
+
+isort $PARENT --check-only \
+              --skip $PARENT/selftests/run \
```

Comment:
Any reason for skipping those `selftests/run` and `unit/test_datadir.py`?

Reply:
They are failing on `isort`, ut as I mentioned, I added all the fails for now without checking them:

```
ERROR: /home/wrampazz/src/avocado/avocado.quick-dev/selftests/run Imports are incorrectly sorted and/or formatted.
ERROR: /home/wrampazz/src/avocado/avocado.quick-dev/selftests/unit/test_datadir.py Imports are incorrectly sorted and/or formatted.
```

Reply:
hum.. my only concern is that we forgot about them.

Reply:
What do you mean? Forgot where?

Reply:
Since we are skipping those files we are moving the errors under the carpet. If those errors are a quick fix, then I prefer to fix then before this PR.

Reply:
Okay, I'll check them and send the fixes here. I was in a rush last week and submitted it the way it is.